src/instructlab/model/evaluate.py

```diff
@@ -191,6 +189,43 @@ def qa_pairs_to_qna_to_avg_scores(qa_pairs: list[dict]) -> dict:
     return qna_to_avg_scores
 
 
+def launch_server(
+    ctx,
+    model,
+    model_name,
+    max_workers,
+    tls_insecure,
+    tls_client_cert,
+    tls_client_key,
+    tls_client_passwd,
```

Comment:
This should probably call `http_client` just like https://github.com/instructlab/instructlab/pull/1373 does it.

Reply:
Done, thanks for the suggestion!